src/GraphQLParser/ROM.cs

```diff
@@ -81,6 +81,8 @@ public override int GetHashCode()
 
         public static implicit operator ReadOnlyMemory<char>(ROM rom) => rom._memory;
 
+        public static implicit operator ReadOnlySpan<char>(ROM rom) => rom._memory.Span;
```

Comment:
For memory on the managed heap, does this create a copy onto the stack?  If yes, maybe it wouldn't be such a great idea to have as an implicit operator.

Reply:
No. `ReadOnlySpan` is a "smart" struct pointing to some chunk of memory - either heap or stack. `Memory.Span` dereference does nothing bad.